### PR TITLE
Add CSV orders output

### PR DESCRIPTION
**Pull request by olegsv**
First version of CSV output for orders subcommand.

### CODE REVIEW

File: src/commands/orders.py [hunk 1]
```diff
@@ -226,8 +229,93 @@ def print_orders_pretty(orders):
...
+
+    price_sell_buy = unformat_amount(format_price(
+      calculate_price(
+          numerator=order['price_numerator'],
+          denominator=order['price_denominator'],
+          decimals_numerator=order['buy_token']['decimals'],
+          decimals_denominator=order['sell_token']['decimals'],
+      )))
+
+    price_buy_sell = unformat_amount(format_price(
```
Comment (anxolin) [inline src/commands/orders.py:268]: Why do you format for later unformat, it doesn't sound right?

File: src/utils/format.py [hunk 2]
```diff
@@ -51,6 +51,8 @@ def format_amount(amount, decimals=18, rounding=ROUND_DOWN):
 
   return f'{rounded_value:,.{decimals}f}'.rstrip('0').rstrip('.')
 
+def unformat_amount(formatted_amount: str) -> float:
```
[the remark this answers is not in the document]
Comment (olegsv) [reply src/utils/format.py:54]: I reused an existing function, and added some output formatting. Not ideal, I know.

File: src/commands/orders.py (hunk 1 above)
Comment (olegsv) [reply src/commands/orders.py]: Fixed this one.

File: src/utils/format.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (olegsv) [reply src/utils/format.py]: Fixed

**Review by anxolin — APPROVED** [review]

**Comment by anxolin** [comment]
Thanks @olegsv !

**Pull request merged**